tti/app/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { Text, View, Image } from "react-native";
+import Login from "./login-componets/login";
+
+
+
+export default function Index() {
+  return (
+    <View
+      style={{
+        flex: 1,
+        justifyContent: "center",
+        alignItems: "center",
+      }}
+    >
```

Comment:
Change to nativewind

Reply:
Changed.. Might need to install dependencies for it to render but Im pretty sure the other team worked to do that already.